vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java

```diff
@@ -268,6 +270,32 @@ private String getDeltaValue() {
         return getElement().getProperty("value");
     }
 
+    /**
+     * Gets the list of colors in HEX format used by the text color picker and
+     * background color picker controls of the text editor.
```

Comment:
Let's mention that this doesn't have a default value and returns null by default. Maybe also elaborate that in this case the web component shows a default palette.

Apart from that modifying the list returned by the method doesn't do anything. Let's return an unmodifyable list which would trigger a respective exception when trying to modify it, and mention in the JavaDoc that this returns an umnodifyable list. There are some similar use cases in the repo already (e.g. setting avatar items).

Reply:
Thanks, updated.